workers/unity/Packages/com.improbable.gdk.mobile/Worker/DefaultMobileWorkerConnector.cs

```diff
@@ -0,0 +1,126 @@
+using System;
+using Improbable.Gdk.Core;
+using Improbable.Worker.CInterop;
+using UnityEngine;
+
+namespace Improbable.Gdk.Mobile
+{
+    public abstract class DefaultMobileWorkerConnector : WorkerConnector
+    {
+        private const string HostIpPlayerPrefsKey = "SpatialOSHostIp";
+
+#pragma warning disable 649
+        [SerializeField] private string IpAddress;
+        [SerializeField] private bool ShouldConnectLocally;
+#pragma warning restore 649
+
+        private void Awake()
+        {
+            InitializeClient();
+        }
+
+        protected virtual void InitializeClient()
+        {
+            var arguments = LaunchArguments.GetArguments();
+            var environment = CommandLineUtility.GetCommandLineValue(arguments, RuntimeConfigNames.Environment, string.Empty);
+            if (string.IsNullOrEmpty(environment))
+            {
+                environment = PlayerPrefs.GetString(RuntimeConfigNames.Environment, string.Empty);
```

Comment:
What if the player prefs isn't set?

Reply:
that's the bug I just encountered on iOS as we don't use command line arguments yet. fixing it right now